command/intake.py

```diff
@@ -0,0 +1,35 @@
+import wpilib
+from robotpy_toolkit_7407.command import SubsystemCommand
+
+import config
+import constants
+from subsystem import Intake
+from robot_systems import Robot
+
+class IntakeEnable(SubsystemCommand[Intake]):
+    def __init__(self, subsystem: Intake):
+        super().__init__(subsystem)
+        self.subsystem = subsystem
+    
+    def execute(self):
+        Robot.intake.intake_enable()
```

Comment:
Move this to the `def initialize` function so that it only runs once. You don't need to run it on every command loop when you only disable the intakes with the IntakeDisable command.